composer.json

```diff
@@ -0,0 +1,80 @@
+{
+    "name": "openeuropa/oe_multilingual",
+    "description": "OpenEuropa Multilingual module.",
+    "type": "drupal-module",
+    "license": "EUPL-1.2",
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "require": {
+        "drupal/administration_language_negotiation": "1.4",
```

Comment:
I just release version 1.5 that contains good stuff, I'd suggest to update to 1.5 directly.

Reply:
We found an issue with @drupol on 1.5, we agreed are gonna stay on 1.4 until that issue is solved, then we upgrade to the next release directly, e.g. 1.6.

Reply:
I've addressed the issue: https://www.drupal.org/project/administration_language_negotiation/issues/2897784#comment-12600524